recipes/libpq/all/conanfile.py

```diff
@@ -270,12 +264,16 @@ def package_info(self):
                 if Version(self.version) >= "12":
                     self.cpp_info.components["pgcommon"].libs.append("pgcommon_shlib")
                     self.cpp_info.components["pgport"].libs = ["pgport", "pgport_shlib"]
+                    if self.settings.os == "Windows":
+                        self.cpp_info.components["pgport"].system_libs = ["ws2_32"]
                     self.cpp_info.components["pgcommon"].requires.append("pgport")
 
         if self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.components["pq"].system_libs = ["pthread"]
         elif self.settings.os == "Windows":
             self.cpp_info.components["pq"].system_libs = ["ws2_32", "secur32", "advapi32", "shell32", "crypt32", "wldap32"]
+            if self.settings.get_safe("compiler.threads") == "posix":
+                self.cpp_info.components["pq"].system_libs.append("pthread")
```

Comment:
it shouldn't be needed

Reply:
The library needs it, no? It calls functions from the threads lib

Reply:
yes but explicit link is not required for MinGW